src/utils/components/PolicyForm/PolicyInterface.tsx

```diff
@@ -308,7 +314,10 @@ const PolicyInterface: FC<PolicyInterfaceProps> = ({
               <Checkbox
                 label={t('Auto-gateway')}
                 id={`policy-interface-gateway-${id}`}
-                isChecked={policyInterface?.ipv4[AUTO_GATEWAY]}
+                isChecked={
+                  policyInterface?.ipv4[AUTO_GATEWAY] === true ||
+                  policyInterface?.ipv4[AUTO_GATEWAY] === undefined
+                }
```

Comment:
This is repeating in this file. Maybe adding some simple helper function that just checks the parameter if it is true or undefined could clear the code a bit.